projectors-core/src/main/java/io/telicent/smart/cache/projectors/sinks/FilterSink.java

```diff
@@ -25,14 +25,20 @@
 import io.telicent.smart.cache.projectors.Library;
 import io.telicent.smart.cache.projectors.Sink;
 import io.telicent.smart.cache.projectors.SinkException;
+import io.telicent.smart.cache.projectors.sinks.builder.AbstractFilteringSinkBuilder;
 import io.telicent.smart.cache.projectors.sinks.builder.AbstractForwardingSinkBuilder;
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.UUID;
+import java.util.function.Function;
```

Comment:
Appears to be unused

Reply:
Cleaned up this and some other unused imports